config/manifests/bases/camel-k.clusterserviceversion.yaml

```diff
@@ -150,11 +150,11 @@ spec:
   - email: users@camel.apache.org
     name: The Apache Software Foundation
   maturity: alpha
-  minKubeVersion: 1.11.0
+  minKubeVersion: 1.14.0
```

Comment:
Do we want to change this? I'd keep it the original version unless there is a reason to change it.

Reply:
Reverted.